src/services/Api.ts

```diff
@@ -68,6 +68,7 @@ const newRequest = <P>(method: HTTP_VERBS, url: string, queryParams: any, data:
 interface LoginRequest {
   username: UserName;
   password: Password;
+  token: Password;
```

Comment:
I think the token could be optional instead of being set to `''` every time, what do you think?